resources/assets/lib/core-legacy/forum.coffee

```diff
@@ -70,15 +70,6 @@ class window.Forum
     parseInt @_totalPostsDiv[0].dataset.total, 10
 
 
-  # null if option not available (not moderator), false/true accordingly otherwise
-  showDeleted: =>
```

Comment:
`forum.topics.delete` blade is still using this; display currently doesn't update properly after deleting/restoring a post